main.tf

```diff
@@ -30,9 +30,10 @@ resource "aws_iam_user_group_membership" "groups" {
 
 # security/password
 resource "random_password" "password" {
-  count   = local.login_on == true ? 1 : 0
-  length  = 16
-  special = true
+  count            = local.login_on == true ? 1 : 0
+  length           = lookup(var.password_policy, "length", 16)
+  special          = true
+  override_special = "!@#$%^&*()_+-=[]{}|'"
```

Comment:
How about below?
```suggestion
  special                = tobool(lookup(var.password_policy, "special", "true"))
  override_special = lookup(var.password_policy, "override_special", "!@#$%^&*()_+-=[]{}|'")
```

Reply:
`override_special` option should not be a variable. It cannot be customized by user. I will add `special` option.